### PR TITLE
Release Sprint 4

### PR DESCRIPTION
**Pull request by MrMatt57**

### CODE REVIEW

File: DOL.WHD.Section14c.Test/Business/Validators/SignatureValidatorTests.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        private ISignatureValidator _signatureValidator;
+        public SignatureValidatorTests()
+        {
+            _signatureValidator = new SignatureValidator();
+        }
+
+        [TestMethod]
+        public void Should_Require_Agreement()
+        {
+            _signatureValidator.ShouldHaveValidationErrorFor(x => x.Agreement, null as bool?);
```
Comment (mgwalker) [inline DOL.WHD.Section14c.Test/Business/Validators/SignatureValidatorTests.cs]: Is there value in also testing `false`?  I know the result should be the same so this is totally acceptable - just curious.

Also, please add tests to verify the cases where there aren't validation errors.

File: DOL.WHD.Section14c.Web/src/modules/components/mainHeaderControl/mainHeaderControlTemplate.html [hunk 2]
```diff
@@ -1,24 +1,36 @@
...
+    <h1>Section 14(c) Certificate Application</h1>
+
+    <nav>
+        <ul>
+            <li><a href="">Dashboard</a></li>
+            <li><a href="" ng-click="vm.userClick()">{{ vm.stateService.user.email ? vm.stateService.user.email : "Username" }}</a></li>
+            <li><a href="" ng-click="vm.saveClick()">Sign Out</a></li>
+        </ul>
+    </nav>
+</header>
```
Comment (mgwalker) [inline DOL.WHD.Section14c.Web/src/modules/components/mainHeaderControl/mainHeaderControlTemplate.html]: I'm really fond of this change.  Cheers for semantic tags!  😃

**Comment by MrMatt57** [comment]
AIS/Sprint 4 release branch is up-to-date and everything is now captured in the PR.

**Comment by stvnrlly** [comment]
I just set up a new account and walked through application submission, testing for 508 issues (using [HTML_Codesniffer](https://squizlabs.github.io/HTML_CodeSniffer/)) along the way. I found 3 sets of them, all in the application itself:

## Employer

- Download links for uploaded files are "found with a valid href attribute, but no link content has been supplied."

## Wage Data

- Download links for uploaded files are "found with a valid href attribute, but no link content has been supplied."

## Work Sites & Employees

- The `.arrow-expander` divs are inside an `<a>` tag "with no link content and no name and/or ID attribute."

**Comment by MrMatt57** [comment]
Updates to address Sprint 4 review are merged and ready for review.

**Comment by rrefoy** [comment]
@MichelleMcNellis Please accept as COR.

**Comment by MichelleMcNellis** [comment]
18F Technical Support & PM have stated this sprint code is accepted. The COR acceptance this code in accordance with the QASP. - 12/14/16 Michelle McNellis

**Pull request merged**